daemon/daemon.go

```diff
@@ -753,6 +753,12 @@ func NewDaemon(config *config.Config, registryService registry.Service, containe
 		return nil, errors.Wrap(err, "couldn't create plugin manager")
 	}
 
+	if len(config.LogConfig.Config) > 0 {
```

Comment:
`config.LogConfig` is already read in two random places in this function (once to log "using default..", and later to set the default).

Could we combine all 3 and extract it into a function `setupDefaultLogConfig()` ?

```go
d.defaultLogConfig, err = setupDefaultLogConfig(config.LogConfig)
if err != nil {
    return nil, err
}
```